gradle-wrapper.properties

```diff
@@ -3,5 +3,5 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
```

Comment:
Wait, we don't need to have `gradle wrapper & its properties` file at the root template folder 🤔 

![image](https://user-images.githubusercontent.com/16315358/201890718-444a9490-5fa8-4e25-bffe-a48f33cb3719.png)

Reply:
@luongvo Maybe, but we will have another PR to remove redundant files/dependencies later.
cc @Tuubz

Reply:
@Tuubz @doannimble I think it's a small task so this is a good time to remove them when we're working on a Gradle upgrading task. What do you think? Do you think we should create a tiny task to remove them or keep them there and we might forget?

Reply:
@doannimble Hmmm, since these files are not related to `template`, I think it's acceptable to do it here 🙏

Reply:
@luongvo We will have a chore ticket to remove other redundant things, not only this one 🙏 
@Tuubz Please create it to avoid forgetting 🙏

Reply:
I'll create an issue for it 👍